tokio/src/runtime/handle.rs

```diff
@@ -101,6 +101,32 @@ impl Handle {
         }
     }
 
+    /// Returns the flavor of the current `Runtime`.
+    ///
+    /// ```
+    /// use tokio::runtime::{Handle, Flavor};
+    ///
+    /// #[tokio::main(flavor = "current_thread")]
+    /// async fn main() {
+    ///   assert_eq!(Flavor::CurrentThread, Handle::current().get_flavor());
+    /// }
+    /// ```
+    ///
+    /// ```
+    /// use tokio::runtime::{Handle, Flavor};
+    ///
+    /// #[tokio::main(flavor = "multi_thread", worker_threads = 4)]
+    /// async fn main() {
+    ///   assert_eq!(Flavor::MultiThread, Handle::current().get_flavor());
+    /// }
+    /// ```
+    pub fn get_flavor(&self) -> Flavor {
+        match self.inner {
+            scheduler::Handle::CurrentThread(_) => Flavor::CurrentThread,
+            scheduler::Handle::MultiThread(_) => Flavor::MultiThread,
+        }
```

Comment:
The order of methods in this file is the order they appear in the documentation. This method should not appear _between_ the `current` and `try_current` methods.

Reply:
Moved it to the end.